tests/src/runtimeApi/device/hipGetDeviceAttribute.cpp

```diff
@@ -109,5 +109,12 @@ int main(int argc, char* argv[]) {
     CHECK(test_hipDeviceGetAttribute(deviceId, hipDeviceAttributeMaxSharedMemoryPerMultiprocessor,
                                      props.maxSharedMemoryPerMultiProcessor));
     CHECK(test_hipDeviceGetAttribute(deviceId, hipDeviceAttributeIntegrated, props.integrated));
+
+    CHECK(test_hipDeviceGetAttribute(deviceId, hipDeviceAttributeMaxTexture1D, props.maxTexture1D));
+    CHECK(test_hipDeviceGetAttribute(deviceId, hipDeviceAttributeMaxTexture2DDimX, props.maxTexture2D[0]));
```

Comment:
Please rename `hipDeviceAttributeMaxTexture2DDimX `and all below attributes to the correct names, as declared in `enum hipDeviceAttribute_t`.

Reply:
Thanks , changed test case to use correct enum values.